src/routes/health.py

```diff
@@ -0,0 +1,21 @@
+"""
+Health check routes for the API.
+"""
+
+from fastapi import APIRouter
+
+from src.service.models import HealthResponse
```

Comment:
I only put models that are part of the business logic rather than just the api in models, but that's up to you